processing/src/main/java/io/druid/segment/filter/TrueFilter.java

```diff
@@ -0,0 +1,74 @@
+/*
+ * Licensed to Metamarkets Group Inc. (Metamarkets) under one
+ * or more contributor license agreements. See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership. Metamarkets licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package io.druid.segment.filter;
+
+import io.druid.query.BitmapResultFactory;
+import io.druid.query.filter.BitmapIndexSelector;
+import io.druid.query.filter.Filter;
+import io.druid.query.filter.ValueMatcher;
+import io.druid.segment.ColumnSelector;
+import io.druid.segment.ColumnSelectorFactory;
+
+/**
+ */
+public class TrueFilter implements Filter
+{
+  public TrueFilter()
+  {
+  }
+
+  @Override
+  public <T> T getBitmapResult(BitmapIndexSelector selector, BitmapResultFactory<T> bitmapResultFactory)
+  {
+    return null;
```

Comment:
Hm, maybe this should return `bitmapResultFactory.wrapAllTrue(Filters.allTrue(selector));` and return `true` for `supportsBitmapIndex`. 

As-is, TrueFilter shouldn't be returning true for `supportsSelectivityEstimation` since it doesn't support bitmap indexes, but not supporting bitmap indexes would potentially interfere with use of bitmap indexes for other subfilters when an And/Or filter contains a TrueFilter (See makeCursors() in QueryableIndexStorageAdapter and getExecutionPlan() in AutoStrategy).